assets/js/modules/analytics-4/components/dashboard/EnhancedMeasurementActivationBanner/InProgressBanner.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * EnhancedMeasurementActivationBanner > InProgressBanner component.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
+import { CircularProgress } from 'googlesitekit-components';
+import BannerNotification from '../../../../../components/notifications/BannerNotification';
+import SuccessGreenSVG from '../../../../../../svg/graphics/ga4-success-green.svg';
+const { useSelect } = Data;
+
+export default function InProgressBanner() {
+	const documentationURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getGoogleSupportURL( {
+			path: '/analytics/answer/9216061',
+		} )
+	);
+
+	return (
+		<BannerNotification
+			id="googlesitekit-enhanced-measurement-activation-banner"
+			className="googlesitekit-enhanced-measurement-success-banner"
+			title={ __( 'Setup in progress', 'google-site-kit' ) }
+			description={ __(
+				'Enhanced measurement is being enabled.',
+				'google-site-kit'
+			) }
+			WinImageSVG={ () => <SuccessGreenSVG /> }
+			format="small"
+			type="win-success"
+			learnMoreLabel={ __( 'Learn more', 'google-site-kit' ) }
+			learnMoreURL={ documentationURL }
+		>
+			<CircularProgress size={ 20 } />
```

Comment:
Showing the spinner icon here looks very odd. Usually it is used in conjunction with something else like next to or within a button and show it only when you click on that button. In our case, if we want to go with the spinner icon, i think it would make more sense to show it in the title of the banner (although this is uncommon pattern for us):

![image](https://github.com/google/site-kit-wp/assets/873049/6fb4d768-258d-40a7-8259-6f7b153f4527)

I think the right element to show here is the progress bar which is usually is used on its own to show users that something happens. In this case, the progress bar will look more naturally and match other UI patterns in our plugin. Let's replace this `CircularProgress` component with the `ProgressBar`.

cc @techanvil

Reply:
Hey guys, this design was approved by [Sigal](https://app.asana.com/0/0/1205707964043448/1205933847705094/f) and [Mariya](https://10up.slack.com/archives/CFFRMC5DE/p1701793154134119), we should check with Sigal in the first instance if we want to change it...

Reply:
Ok, let's stick with this solution then.